lib/packagekit-glib2/pk-client.c

```diff
@@ -194,6 +194,8 @@ pk_client_state_unset_proxy (PkClientState *state)
 						      state);
 		g_clear_object (&state->proxy);
 	}
+
+	g_object_unref (state);
```

Comment:
This implies that `pk_client_state_unset_proxy()` takes ownership of `state`, which doesn’t fit in with common conventions for ownership transfer (only `unref` or `steal` functions should take ownership).